### PR TITLE
Add my side about use sys.getsizeof

### PR DESCRIPTION
**Pull request by YibiChen**
for today's meeting about python tips

### CODE REVIEW

**Comment by luispedro** [comment]
You need to resolve the conflicts manually before it can be accepted

File: talk.md [hunk 1]
```diff
@@ -908,5 +910,40 @@ In[20]:     %%prun
 
      ....
 ```
+## 24: Use sys.getsizeof to guide data struction optimization (_Yibi_)
```
Comment (luispedro) [inline talk.md:914]: I believe you are missing a separator (`---`) here

**Review by luispedro — CHANGES_REQUESTED** [review]
See comment about separator

**Comment by YibiChen** [comment]
Fixed. Thanks.

**Pull request merged**